### ISSUE TITLE
mega-reads exited before assembly

### ISSUE THREAD

**Issue by HegedusB** [alekseyzimin/masurca#117]
Hi! I´m trying to run Masurca with Nanopore and Illumina PE reads. The program started correctly, but after a short time I get this error massage:

“error reading mega-reads file at /opt/Masurca/MaSuRCA-3.3.3/bin/find_contained_reads.pl line 33, <FILE> line 82867.
[Tue Jun 25 08:50:48 EDT 2019] failed to create mega-reads frg file
[Tue Jun 25 08:50:48 EDT 2019] mega-reads exited before assembly”

What do you think? What went wrong.
Thanks!

**Comment by milady81** [comment]
> Hi! I´m trying to run Masurca with Nanopore and Illumina PE reads. The program started correctly, but after a short time I get this error massage:
> 
> “error reading mega-reads file at /opt/Masurca/MaSuRCA-3.3.3/bin/find_contained_reads.pl line 33, line 82867.
> [Tue Jun 25 08:50:48 EDT 2019] failed to create mega-reads frg file
> [Tue Jun 25 08:50:48 EDT 2019] mega-reads exited before assembly”
> 
> What do you think? What went wrong.
> Thanks!

I have exactly the same problem. 
#HegedusB did you solve the problem?

I see no answer from the MaSuRCA owner, hopefully will help us.

**Comment by HegedusB** [comment]
No. Not really. I am reading the forums comments, but have not found any meaningful help yet. I am using it on a clean debian linux, in a docker environment. So I am working as clean as possible. I have used the reads with the canu assembler without any problem. I think, the read quality can not be the problem. Now I am really curious.

**Comment by milady81** [comment]
Hi, Actually I have a problem with both assemblers: Canu and Flye.
Can: I have an error as you have and using Flye: 

[2019-06-26 10:33:21] INFO: Starting Flye 2.4.1-release
[2019-06-26 10:33:21] INFO: >>>STAGE: configure
[2019-06-26 10:33:21] INFO: Configuring run
[2019-06-26 10:33:22] ERROR: Invalid char while reading mr.41.15.15.0.02.1.fa

I check the mr.41.15.15.0.02.1.fa and do not see any invalid characters?!
Maybe you have any idea where is a problem? 

ps. I do not think the read quality is an issue with our case. I am very curious also, what solution we could get.

**Comment by JuliaVoe** [comment]
@HegedusB and @milady81 , I'm running into the same problem with "error reading mega-reads file at usr/MaSuRCA-3.3.3/bin/find_contained_reads.pl line 33". So far, I couldn't find any solutions. Did you eventually solve this issue?
I checked some files, work1_mr1/readPlacementsInSuperReads.final.read.superRead.offset.ori.txt and mr.41.15.17.0.029.1.fa are present. I also checked for containees.txt, which is created in the mega_reads_assemble_cluster.sh script in line 679 (find_contained_reads.pl work1_mr1/readPlacementsInSuperReads.final.read.superRead.offset.ori.txt $COORDS.1$POSTFIX.fa > containees.txt). The containees.txt file is created, but it's empty. So something must go wrong there, however, I can't figure out what it is...
Any help is highly appreciated :)

**Comment by gitcruz** [comment]
Hi all, 

I am having the same problem but it coincides with an I/O problem that affected our server last night. I am removing the likely corrupted files: mr.41.15.15.0.02.1.fa and mr.fa.in

By the way, did any of you also registered errors at the superreads creation step? as I reported here: https://github.com/alekseyzimin/masurca/issues/184

I think is totally unrelated but who knows.

Best,
Fernando
